gcloud-java-storage/src/main/java/com/google/gcloud/storage/testing/FakeStorageRpc.java

```diff
@@ -41,12 +40,6 @@ public FakeStorageRpc(boolean throwIfOption) {
     this.throwIfOption = throwIfOption;
   }
 
-  // remove all files
-  void reset() {
-    stuff = new HashMap<>();
-    contents = new HashMap<>();
-  }
-
```

Comment:
`FakeStorageRpc` is now a private final field on each test class. So it's recreated with empty contents before each test method executes. Since this PR migrates the tests to not use the SPI at all, we're able to guarantee the library is using the instance stored in the test.